modules/profiles/laptop-x86.nix

```diff
@@ -112,6 +112,10 @@ in {
       profiles = {
         applications.enable = false;
       };
+
+      # Logging services
+      logging.client.enable = true;
+      logging.server.enable = true;
     };
```

Comment:
The cloud grafana logging server ip declaration could go here

Reply:
Added server address declaration.